undefined_syms.txt

```diff
@@ -1539,10 +1539,27 @@ D_06009E70 = 0x06009E70;
 
 // ovl_En_Akindonuts
 
+D_06000994 = 0x06000994;
+D_060012F4 = 0x060012F4;
 D_06001350 = 0x06001350;
+D_06001BC8 = 0x06001BC8;
+D_06001E2C = 0x06001E2C;
+D_06002268 = 0x06002268;
+D_06002670 = 0x06002670;
+D_060029E8 = 0x060029E8;
+D_06002F08 = 0x06002F08;
+D_06003438 = 0x06003438;
+D_060038CC = 0x060038CC;
+D_06003CC0 = 0x06003CC0;
+D_06004700 = 0x06004700;
+D_06004AA0 = 0x06004AA0;
+D_06004E38 = 0x06004E38;
 D_06005488 = 0x06005488;
+D_0600577C = 0x0600577C;
+D_06005CA8 = 0x06005CA8;
 D_06008290 = 0x06008290;
 D_0600AC70 = 0x0600AC70;
+D_0600B0B4 = 0x0600B0B4;
```

Comment:
Should be able to remove these now right?

Reply:
And the ones from EnTru, which apparently I forgot to check.